presto-hudi/pom.xml

```diff
@@ -216,6 +216,11 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
```

Comment:
Should it be part of hudi-presto-bundle itself?

Reply:
I just think so, but actually not. So i have to add it explicitly.